### PR TITLE
Support specify cluster_id without prefix

### PR DESCRIPTION
**Pull request by cs-zhang**
Port from https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/187

Signed-off-by: CS Zhang <zhangcho@us.ibm.com>

### CODE REVIEW

File: docs/var.tfvars-doc.md
Comment (yussufsh) [inline docs/var.tfvars-doc.md:191]: cluster_if_prefix->cluster_id_prefix

Comment (cs-zhang) [reply docs/var.tfvars-doc.md]: fixed

**Comment by yussufsh** [comment]
@cs-zhang Could you please squash the commits into one?

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/241#" title="Author self-approved">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/241#pullrequestreview-617164017" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/241#pullrequestreview-617174202" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/241#" title="Author self-approved">cs-zhang</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/241#pullrequestreview-617164017" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
